basis/basis_ffi.c

```diff
@@ -143,7 +143,7 @@ void ffiopen_out (unsigned char *c, long clen, unsigned char *a, long alen) {
   #ifdef EVAL
   int fd = open((const char *) c, O_RDWR|O_CREAT|O_TRUNC, S_IRUSR|S_IWUSR|S_IRGRP|S_IROTH);
   #else
-  int fd = open((const char *) c, O_RDWR|O_CREAT|O_TRUNC);
+  int fd = open((const char *) c, O_RDWR|O_CREAT|O_TRUNC, 0600);
```

Comment:
I think it would be better to replace this whole chunk with what's on line 144. I checked how `TextIO.openOut` works with Poly/ML and Mlton on my machine, and both seem consistent with this (i.e., -rw-rw-r--).